app/Tournament.js

```diff
@@ -0,0 +1,42 @@
+
+import {Round} from './Round.js';
+import {shuffle} from '../utils/common.js'
+export class Tournament{
+  constructor(Players){
+    this._Players=Players;
+    this._tournamentRounds=[];
+  }
+  get Players(){
+    return this._Players;
+  }
+  get tournamentRounds(){
+    return this._tournamentRounds;
+  }
+  set tournamentRounds(tournamentRounds){
+    return this._tournamentRounds=tournamentRounds;
+  }
+  maketheRounds(){
+    for(var i=this._Players.length;i>1;i=parseInt(i/2)){
```

Comment:
Don't use parseInt to round. Parse means converting a string to a number. Math.floor?